fe/fe-core/src/main/java/com/starrocks/analysis/SingleRangePartitionDesc.java

```diff
@@ -145,6 +152,16 @@ public void analyze(int partColNum, Map<String, String> otherProperties) throws
 
         tabletType = PropertyAnalyzer.analyzeTabletType(properties);
 
+        // analyze enable storage cache and cache ttl
+        boolean enableStorageCache = PropertyAnalyzer.analyzeBooleanProp(
+                properties, PropertyAnalyzer.PROPERTIES_ENABLE_STORAGE_CACHE, false);
+        long storageCacheTtlS = PropertyAnalyzer.analyzeLongProp(
+                properties, PropertyAnalyzer.PROPERTIES_STORAGE_CACHE_TTL, 0);
+        if (enableStorageCache && storageCacheTtlS == 0) {
+            throw new AnalysisException("Storage cache ttl should not be 0 when cache is enabled");
```

Comment:
Could the `ttl == 0` means forever exists without cleanup cache?

Reply:
0 indicates "never cache", like presto metacache, jdk networkaddress.cache.ttl.
-1 indicates "cache forever" is better.